ui/lib/core/addon/helpers/has-feature.js

```diff
@@ -12,10 +12,12 @@ const FEATURES = [
   'Seal Wrapping',
   'Control Groups',
   'Namespaces',
+  'KMIP',
```

Comment:
Nice!

Reply:
It looks like it was an oversight, because the places where we check for KMIP prior to this were in the component files, not on the template. But that's a great reminder to double-check that this doesn't have unintended side-effects